.github/workflows/causal_lm_cpp.yml

```diff
@@ -723,6 +723,19 @@ jobs:
           && ./build/samples/cpp/visual_language_chat/visual_language_chat ./llava_1_5_7b_ov/ monalisa.jpg
           <<< $'Who drew this painting?\nWhen did the painter live?'
         timeout-minutes: 4
+      - name: Download and convert LLaVa-Next model and an image
+        run: |
+          source ./ov/setupvars.sh
+          python -m pip install ./thirdparty/openvino_tokenizers/[transformers] --pre --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
+          python -m pip install --upgrade-strategy eager -r ./samples/requirements.txt --pre --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
```

Comment:
dependencies are already installed above (twice)
maybe we can split install dependencies step and then download models minicpm, LLaVA 1.5, LLaVa next

Reply:
Moved to separate step